1 - identify organisations in the corpus/ner_flair.ipynb

```diff
@@ -1,25 +1,59 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from flair.models import SequenceTagger
</pre><p><code>flair</code> is currently not in <code>requirements.txt</code>	! [Required]</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/disaster-capitalism/named-entities/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
okay added.